### PR TITLE
Eliminate confusion

### PR DESCRIPTION
**Pull request by HamDerAndrew**
I didn't realise I had to do the "git add" command BEFORE running the commands. Adding that to prevent others from doing the same.

## Description

_What did you add/update/change?_

## Type of suggestion

* [x] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

**Comment by HamDerAndrew** [comment]
I would also like to add that the commands for MacOS are a bit different than for Windows users (docs is for Windows I noticed).

Check this StackOverflow response:
https://stackoverflow.com/a/77053318

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ The zip file includes the following files:
 
 13. Give permission for these scripts to be able to run within the workflows > scripts folder using the following commands in the Command Line:
 
+{% hint style="info" %}
+Ensure you run **git add name-of-file.sh** before running the below commands to add untracked files to the git index.
```
Comment (alina-tincas) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md:58]:
```suggestion
Ensure you run **git add name-of-file.sh** before running the below commands on Windows to add untracked files to the git index.
```

**Review by alina-tincas — APPROVED** [review]
Heya @HamDerAndrew thank you for the PR 😁 

I have added "on windows" in the text to make it more obvious it is the windows commands in this case. 

Merging this in!

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md [hunk 2]
```diff
@@ -54,15 +54,15 @@ The zip file includes the following files:
 
 13. Give permission for these scripts to be able to run within the workflows > scripts folder using the following commands in the Command Line:
 
+{% hint style="info" %}
+Ensure you run **git add name-of-file.sh** before running the below Windows commands to add untracked files to the git index.
+{% endhint %}
```
Comment (alina-tincas) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/github-actions.md:59]:
```suggestion
{% hint style="info" %}

Ensure you run **git add name-of-file.sh** before running the below Windows commands to add untracked files to the git index.

{% endhint %}
```

**Review by alina-tincas — APPROVED** [review]
fix lint warning

**Comment by alina-tincas** [comment]
Happy new year @HamDerAndrew 🎉 

There is actually a new PR where now it is gonna be only 1 page for azure devops and another for github actions each with an example using bash and powershell scripts. The example from this page you are trying to make the changes is gonna be overwritten with new examples created by the CI/CD team, therefore I am closing this PR and once it is merged you can give those examples a try 😁

**Pull request closed without merging**